src/ipc/ipc3/dai.c

```diff
@@ -121,12 +121,9 @@ int ipc_dai_data_config(struct comp_dev *dev)
 		dd->config.burst_elems = dai_config->ssp.tdm_slots;
 		break;
 	case SOF_DAI_INTEL_DMIC:
-		/* We can use always the largest burst length. */
-		dd->config.burst_elems = 8;
-
-		comp_info(dev, "config->dmic.fifo_bits = %u config->dmic.num_pdm_active = %u",
-			  dai_config->dmic.fifo_bits,
-			  dai_config->dmic.num_pdm_active);
+		/* Depth is passed by DMIC driver that retrieves it from blob */
```

Comment:
change ipc3/dai.c or ipc4/dai.c ?

Reply:
This change is needed also to IPC4, thanks for pointing that!

Reply:
Done